tensorly/decomposition/_cp.py

```diff
@@ -73,8 +74,19 @@ def initialize_cp(tensor, rank, init='svd', svd='numpy_svd', random_state=None,
 
     elif isinstance(init, (tuple, list, CPTensor)):
         # TODO: Test this
-        try:
+        try:         
+            if normalize_factors is True:
+                warnings.warn('It is not recommended to initialize an initial tensor with normalizing. Consider normalizing the initial tensor before using this function')
```

Comment:
We will clarify the warning indeed.

The behavior we wanted to avoid was 
1. the user inputs a tensor with non-unit weights
2. the weights are pulled on the last factor
3. the factors are normalized
In that case I was worried that the weights pulling was useless and that could confuse the user, therefore suggesting to normalize beforehand. Indeed a user might want to normalize his input initialization, but then is it not a better practice to do it outside the init function, so that the init function does not change the init tensor itself? To me the normalization option should only be allowed for random or svd init.

Reply:
Yes, it's good to warn the user that their normalization will be overwritten. But that will anyway right? I don't think the user expects the initialization to not change? I tend to just view (weights, factors) as a generic representation in CP form rather than a normalized CP. They may have gotten the (weights, factors) from e.g. a previous run of CP with normalize=True and just want to do warm restart. 

I'm actually curious about the actual impact of normalization on stability within CP.